packages/plugin/src/message-type-extensions/google-types.ts

```diff
@@ -8,14 +8,15 @@ import {
 import * as ts from "typescript";
 import {LongType} from "@protobuf-ts/runtime";
 import {CustomMethodGenerator} from "../code-gen/message-type-generator";
+import { FieldInfoGenerator } from "src/code-gen/field-info-generator";
```

Comment:
TypeScript sometimes suggests importing from `src` in this monorepo, but `src` no longer exists when the package is built. This will fix the CI failure:

```suggestion
import { FieldInfoGenerator } from "../code-gen/field-info-generator";
```